### DISCUSSION TITLE
Crash Course

### DISCUSSION THREAD

**Discussion by Sthaaz** [OrchardCMS/OrchardCore#7339] [Q&A]
Can we have a crash course for Orchard Core to get started for new developers?

**Comment by jptissot** [comment]
I think there were workshops previously held. Not sure when the next one would be. Maybe if there is enough demand for it we will have other ones ? https://www.orchardcore.net/workshops

**Reply by Sthaaz** [reply]
Well, it was a webinar workshop not everyone can or join in. Is there any other like on YouTube where it is easily accessible.

**Reply by jptissot** [reply]
You could  check out Lombiq's youtube channel. https://www.youtube.com/channel/UCDVUxCz2RvkgTbA0wAYKwRA
